### DISCUSSION TITLE
About using tags

### DISCUSSION THREAD

**Discussion by olegprogs** [netbox-community/netbox#9447] [Q&A]
What are the pros and cons of using tags in a netbox?
Can there any negative impact of their use on netbox operation now or in the future? 
Will there be any problems when upgrading to new versions related to tags?
I have many tenants within the same group. I want to tag two subgroups with tags. 
But the admins say that this is a bad idea and it's better not to use tags at all. 
Is there a better solution for this case? (I don't want to split these tenants into many small groups in this case.)

**Comment by kkthxbye-code** [comment]
> I have many tenants within the same group. I want to tag two subgroups with tags.
> But the admins say that this is a bad idea and it's better not to use tags at all.

Tenant groups can be nested, just create the subgroups as tenant groups with the existing group as parent.

**Reply by olegprogs** [reply]
Thanks for the quick response.
What do you think about using tags?
Is it really better not to use them?

**Comment by Kludge-As-A-Service** [comment]
Ask them why they think it's better not to use tags. In itself, tagging is just a feature (that is present in a lot of modern tools), there is nothing wrong with it.
